libs/sdk/src/NativeBridge.ts

```diff
@@ -40,7 +40,11 @@ export interface Bridge {
     traits: Record<string, unknown> | null,
     options: Record<string, unknown> | null,
   ): Promise<void>;
-  alias(newId: string, options: Record<string, unknown> | null): Promise<void>;
+  alias(
```

Comment:
We should define overloaded API here and use it in the implementation.

Reply:
This is required for the API bridge and should correspond to the API defined in the Android and iOS modules.

If you have any other suggestions, please let me know.

Reply:
The bridge can still support overloaded APIs. In the implementation, the functions can set `null` values for appropriate arguments while invoking the native SDK modules.

Now, you're setting `null` for the arguments in `RudderEvents.tsx`.